Zadanie 11.2 Użycie 'Self'/js/scripts.js

```diff
@@ -0,0 +1,21 @@
+//scripts.js
+
+function Button(text) {
+    this.text = text || 'Hello';
+}
+
+Button.prototype = {
+    create: function() {
+        var btn1 = new Button('Hello');
+        var self = this;
+        this.$element = $('<button>');
+        this.$element.text(this.text);
+        this.$element.click(function() {
+            alert(self.text);
+            $('body').append(this.$element);
```

Comment:
ten append jest w zlym miejscu - na co wskazuje tutaj `this`? (mozesz zrobic sobie `console.log(this);`)